pkg/cmd/cli/restore/logs.go

```diff
@@ -82,6 +83,12 @@ func (l *LogsOptions) Validate(f client.Factory) error {
 	}
 	l.client = c
 
-	_, err = l.client.ArkV1().Restores(f.Namespace()).Get(l.RestoreName, metav1.GetOptions{})
+	r, err := l.client.ArkV1().Restores(f.Namespace()).Get(l.RestoreName, metav1.GetOptions{})
+	if err != nil {
+		return err
+	}
+	if r.Status.Phase != v1.RestorePhaseCompleted {
+		err = errors.Errorf("unable to retrieve logs because restore is not complete")
```

Comment:
`errors.New(...)` since it's not a format string, and I'd probably just return the error here and change L93 to a return nil.

Reply:
+1 to returning here

I'm almost always in favor of Errorf 100% of the time even if there aren't format strings - saves you the time of having to replace New with Errorf if you do decide you need formatted vars